LR: Modificación del ranking para que sea semanal-#184

javsolgar opened the pull request:
Se ha modificado el ranking para que muestre los usuarios con una mayor puntuación semanal.

javsolgar committed:
Modificado ranking-#184

Se ha modificado el ranking para que sea semanal

jorandmol reviewed (approved):
Funciona a la perfección, buen trabajo. Además, no se han detectado nuevas incidencias de Codacy.

jorandmol reviewed (changes requested):
La funcionalidad está correcta y la he aprobado, pero me he dado cuenta que al cambiar el voto se suma a los puntos semanales en vez de actualizar este valor.

javsolgar committed:
Solucionado fallo puntuacionSemanal-#184

Se ha solucionado fallo al restar puntuaciones y se han actualizado test para comprobar el correcto funcionamiento de esto.

javsolgar commented:
@jorandmol tenías razón, había un fallo al actualizar la puntuación semanal cuando se cambiaba una valoración. Se ha arreglado y se han actualizado los test para comprobar que este fallo no vuelva a ocurrir.

jorandmol reviewed (approved):
Se ha comprobado los cambios y funciona como se esperaba
